index.html

```diff
@@ -10,50 +10,61 @@
-            </li>
-            <li>
-            </li>
-            <li>
-                <a href=""><img src="/images/iconmonstr-medium-1.svg"></a>
-            </li>
-            <li>
-            </li>
-            <li>
-                <a href="https://github.com/stephenkati"><i class="fa-brands fa-github"></i></a>
-            </li>
-            <li>
-            </li>
-            <li>
-                <a href=""><i class="fa-brands fa-angellist"></i></a>
-            </li>
-        </ul>
-       
+    <div class="top-container">
+        <header>
+            <div class="toggle-menu">
+                <h3 class="welcome"><a href="">Welcome</a></h3>
+                <nav class="small">
+                    <i class="fa-solid fa-bars"></i>
+                </nav>
+                <nav class="desktop-nav">
+                    <ul>
+                        <li><a href="#" >Portfolio</a></li>
+                        <li><a href="#About" >About</a></li>
+                        <li><a href="#Contact" >Contact</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <div class="contain">
+    
+            <section class="intro">
+                <h1>Hey there. I’m Stephen. <br/><span>I’m a software developer</span></h1>
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </section>
+    
+            <div class="contacts">
+                <ul class="icons">
+                    <li>
+                        <a href="https://twitter.com/Stephen57913145"><i class="fa-brands fa-twitter"></i></a>
+                    </li>
+                    <li>
+                    </li>
+                    <li>
+                        <a href="https://www.linkedin.com/in/stephen-katuli-a92752251/"><i class="fa-brands fa-linkedin"></i></a>
+                    </li>
+                    <li>
+                    </li>
+                    <li>
+                        <a href="" class="medium"><img src="/images/iconmonstr-medium-1.svg"></a>
+                    </li>
+                    <li>
+                    </li>
+                    <li>
+                        <a href="https://github.com/stephenkati"><i class="fa-brands fa-github"></i></a>
+                    </li>
+                    <li>
+                    </li>
+                    <li>
+                        <a href=""><i class="fa-brands fa-angellist"></i></a>
+                    </li>
+                </ul>
+            </div>
+            <i class="fa-solid fa-angle-down"></i>
+        </div>
     </div>
-</div>
+    
     <section class="work">
-        <i class="fa-solid fa-angle-down"></i>
         <h1 class="work-title">My Recent Works</h1>
```

Comment:
- [ ] It's important that the website looks good across all screen sizes 💯 For the works section, it's better if the number of the projects cards depends on the width available in the screen, for that you can use `flexbox` then use `wrap` 👇 same thing for the skills section, both sections are cropped in smaller screen sizes because the width is not enough for 3 cards 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/213429736-985072dc-098f-47c7-975b-03594c60cf4b.png)
![image](https://user-images.githubusercontent.com/50721479/213429982-4acb49bf-a9be-414a-ae15-0c9809316e2d.png)